src/plopp/graphics/colormapper.py

```diff
@@ -227,9 +227,10 @@ def _set_normalizer_limits(self):
             self.normalizer.vmin = self.vmin
             self.normalizer.vmax = self.vmax
 
-    def update(self, key: str, data: sc.DataArray):
+    def update(self, _=None, **kwargs):
```

Comment:
The argument name confused me here, I always assume `_` is unused, so I was surprised when it was used.
Maybe we should rename it to `values`?

Reply:
I basically wanted to avoid a name clash from happening. Like when you update a dict in python.
You can do
```Py
d.update(a=1, b=3, values=5)
d.update({'a': 1, 'b': 3, 'values': 5})
```
If my function signature is
```Py
def update(self, values=None, **kwargs):
```
then the first case will fail because it is expecting `values` to be a dict, not just a single value.

I don't actually know how python does it, because you can also use `_` as a key:
```Py
d = {'a': 1, 'b': 3}
d.update(_=55)
print(d)
d.update({'a': 33, 'c': 6})
print(d)
```
```
{'a': 1, 'b': 3, '_': 55}
{'a': 33, 'b': 3, '_': 55, 'c': 6}
```
Maybe I should look at how they implemented it...

Reply:
So I think a better way of implementing this is
```Py
def update(self, *args, **kwargs):
    new = kwargs
    if args:
        if len(args) > 1:
            raise TypeError('Update expected at most 1 arg.  Got {}.'.format(len(args)))
        else:
            new.update(args[0])
    ...
```

Reply:
No I think what you did makes sense. We can avoid clashes and make the name more clear by using something like: `_dict_arg`. That's unlikely to cause a clash and it's also a bit more clear. But I'll leave it to you to decide.

Reply:
Even with `*args` there is no clash, you can still do `view.update(args=1, a=2)` and the keys will be `args` and `a`.